Jenkinsfile

```diff
@@ -1,3 +1,5 @@
+buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```

Comment:
By adding the usual build configuration, we test things on various recommended configurations.
By leaving the Pipeline code below written by Félix, we still run the special `TheFlintstonesTest` testcase inside the custom environment, so we keep covering the whole testing picture.